new-lamassu-admin/src/pages/Assets/Assets.js

```diff
@@ -0,0 +1,213 @@
+            </TableHead>
+            <TableBody>
+              {data?.map((asset, idx) => {
+                if (idx < numToRender) {
+                  return (
+                    <TableRow className={classes.row} key={idx}>
+                      <Cell align="left">
+                        <P>{asset.display}</P>
+                      </Cell>
+                      <Cell align="right">
+                        <P>{`${selectAmountPrefix(asset)}
+                         ${formatCurrency(Math.abs(asset.amount))} ${
+                          asset.currency
+                        }`}</P>
+                      </Cell>
+                    </TableRow>
+                  )
+                }
+                return null
+              })}
+              <TableRow className={classes.totalRow} key={data?.length + 1}>
+                <Cell align="left">
+                  <Info2>{`Total ${R.toLower(title)}`}</Info2>
+                </Cell>
+                <Cell align="right">
+                  <Info2>{`${formatCurrency(totalAmount)} ${currency}`}</Info2>
+                </Cell>
+              </TableRow>
+            </TableBody>
+          </Table>
+        </TableContainer>
+      </Grid>
+    </>
+  )
+}
+
+const formatCurrency = amount =>
+  amount.toLocaleString('en-US', { maximumFractionDigits: 2 })
+
+const Assets = () => {
+  const classes = useStyles()
+
+  const mockData = [
+    {
+      id: 'fiatBalance',
+      display: 'Fiat balance',
+      amount: 10438,
+      currency: 'USD',
+      class: 'Available balance'
+    },
+    {
+      id: 'hedgingReserve',
+      display: 'Hedging reserve',
+      amount: -1486,
+      currency: 'USD',
+      class: 'Available balance',
+      direction: 'out'
+    },
+    {
+      id: 'hedgedWalletAssets',
+      display: 'Hedged wallet assets',
+      amount: 96446,
+      currency: 'USD',
+      class: 'Wallet assets',
+      direction: 'in'
+    },
+    {
+      id: 'unhedgedWalletAssets',
+      display: 'Unhedged wallet assets',
+      amount: 3978,
+      currency: 'USD',
+      class: 'Wallet assets',
+      direction: 'in'
+    }
+  ]
+
+  const mockDataTotal = [
+    {
+      id: 'fiatBalance',
+      display: 'Fiat balance',
+      amount: 10438,
+      currency: 'USD'
+    },
+    {
+      id: 'hedgingReserve',
+      display: 'Hedging reserve',
+      amount: -1486,
+      currency: 'USD',
+      direction: 'out'
+    },
+    {
+      id: 'hedgedWalletAssets',
+      display: 'Market value of hedged wallet assets',
+      amount: 94980,
+      currency: 'USD',
+      direction: 'in'
+    },
+    {
+      id: 'unhedgedWalletAssets',
+      display: 'Unhedged wallet assets',
+      amount: 3978,
+      currency: 'USD',
+      direction: 'in'
+    }
+  ]
+
+  const filterByClass = x =>
+    R.filter(it => R.path(['class'])(it) === x)(mockData)
+
+  return (
+    <>
+      <TitleSection title="Balance sheet" />
+      <div className={classes.root}>
+        <Grid container>
```

Comment:
Isn't this Grid element redundant? As you already have the wrapping div

Reply:
Hmm, I think not, if removed the tables will be displayed in a single column.